app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarIntegration.kt

```diff
@@ -42,6 +42,7 @@ abstract class ToolbarIntegration(
         toolbar,
         store,
         sessionId,
+        context.settings().showUnifiedSearchFeature,
```

Comment:
I faced this issue on a local build with autoPublish A-C. I guess this build fails as A-C is still not released with these changes --> https://github.com/mozilla-mobile/firefox-android/pull/294/files#diff-b86aeb4d890cf642393212771d4a167730c8526d6761960a9226a94ae198b7f9 

Now I'm curious if t the A-C version bump merge will fail on Fenix since main branch here has 4 params, guess named params will solve the issue. Should we add the change with named params before so the A-C merge doesn't fail?

Reply:
AC is published based on a cron job twice a day and then a PR to merge the new version in Fenix is automatically opened.
In cases of breaking A-C changes we usually:
- wait for the next PR to bump the A-C version
- close that PR with a comment explaining that the bump will be made in another ticket that addresses the breaking change
- add a new commit to the PR that addresses the breaking change to bump the AC version

Such that when the PR that addresses the breaking change (like this PR) is merged it will also include the A-C version bump.